src/test/java/org/jetbrains/bsp/bazel/BazelBspServerTestData.java

```diff
@@ -119,41 +129,88 @@
                           ImmutableList.of())))));
 
   static final ScalaTestClassesParams SCALA_TEST_CLASSES_PARAMS =
-      new ScalaTestClassesParams(ImmutableList.of(ID_1, ID_4));
+      new ScalaTestClassesParams(
+          ImmutableList.of(EXAMPLE_EXAMPLE_TARGET, EXAMPLE_EXAMPLE_TEST_TARGET));
 
   static final ScalaTestClassesResult EXPECTED_SCALA_TEST_CLASSES =
       new ScalaTestClassesResult(
           ImmutableList.of(
-              new ScalaTestClassesItem(ID_4, ImmutableList.of("example.ExampleTest"))));
+              new ScalaTestClassesItem(
+                  EXAMPLE_EXAMPLE_TEST_TARGET, ImmutableList.of("example.ExampleTest"))));
+
+  static final JavacOptionsParams JAVAC_OPTIONS_PARAMS =
+      new JavacOptionsParams(ImmutableList.of(EXAMPLE_EXAMPLE_TARGET, DEP_JAVA_DEP_TARGET));
+
+  static final JavacOptionsResult EXPECTED_JAVAC_OPTIONS =
+      new JavacOptionsResult(
+          ImmutableList.of(
+              new JavacOptionsItem(
+                  EXAMPLE_EXAMPLE_TARGET, ImmutableList.of(), ImmutableList.of(), ""),
+              new JavacOptionsItem(
+                  DEP_JAVA_DEP_TARGET,
+                  ImmutableList.of("-Werror", "-Xlint:all"),
+                  ImmutableList.of(),
+                  "")));
+
+  static final ScalacOptionsParams SCALAC_OPTIONS_PARAMS =
+      new ScalacOptionsParams(ImmutableList.of(EXAMPLE_EXAMPLE_TARGET, DEP_DEP_TARGET));
+
+  static final ScalacOptionsResult EXPECTED_SCALAC_OPTIONS =
+      new ScalacOptionsResult(
+          ImmutableList.of(
+              new ScalacOptionsItem(
+                  EXAMPLE_EXAMPLE_TARGET,
+                  ImmutableList.of("-target:jvm-1.8"),
+                  ImmutableList.of(
+                      "__main__/external/io_bazel_rules_scala_scala_library/scala-library-2.12.8.jar",
+                      "__main__/external/io_bazel_rules_scala_scala_reflect/scala-reflect-2.12.8.jar",
+                      "__main__/bazel-out/darwin-fastbuild/bin/external/io_bazel_rules_scala/src/java/io/bazel/rulesscala/scalac/scalac.jar",
+                      "__main__/bazel-out/darwin-fastbuild/bin/dep/deeper/deeper.jar",
+                      "__main__/bazel-out/darwin-fastbuild/bin/dep/dep.jar"),
+                  "__main__/bazel-out/darwin-fastbuild/bin/example/"),
+              new ScalacOptionsItem(
+                  DEP_DEP_TARGET,
+                  ImmutableList.of(),
+                  ImmutableList.of(
+                      "__main__/external/io_bazel_rules_scala_scala_library/scala-library-2.12.8.jar",
+                      "__main__/external/io_bazel_rules_scala_scala_reflect/scala-reflect-2.12.8.jar",
+                      "__main__/bazel-out/darwin-fastbuild/bin/external/io_bazel_rules_scala/src/java/io/bazel/rulesscala/scalac/scalac.jar",
+                      "__main__/bazel-out/darwin-fastbuild/bin/dep/deeper/deeper.jar"),
+                  "__main__/bazel-out/darwin-fastbuild/bin/dep/")));
```

Comment:
Pls remove the `__main__/bazel-out/darwin-fastbuild` from the beggining of the paths. `darwin-fastbuild` is a path that's specifically created in OSX. Other OS's use different names :smile:

Reply:
ohh righ